SitumWayfinding/Classes/Helpers/UIUtils.swift

```diff
@@ -46,6 +46,48 @@ extension UIView {
         self.bottomAnchor.constraint(equalTo: superview.bottomAnchor, constant: 0).isActive = true
         self.leadingAnchor.constraint(equalTo: superview.leadingAnchor, constant: 0).isActive = true
         self.trailingAnchor.constraint(equalTo: superview.trailingAnchor, constant: 0).isActive = true
-        
+    }
+    
+    func roundCorners(corners: UIRectCorner) {
+        let radius: CGFloat = 5
+        if #available(iOS 11.0, *) {
+            self.layer.maskedCorners = rectCornerToMaskedCorners(corners: corners)
+            self.layer.cornerRadius = radius
+        } else {
+            let path = UIBezierPath(roundedRect: self.bounds,
+                byRoundingCorners: corners,
+                cornerRadii: CGSize(width: radius, height: radius))
+            let mask = CAShapeLayer()
+            mask.path = path.cgPath
+            // use UIRectCorner with UIBezierPath to clip view to be able to round corners
+            self.layer.mask = mask
+        }
+    }
+    
+    /**
+     This function is made to easily use UIRectCorner always to specify which corners are rounded in a view.
+     UIRectCorner is a clearer OptionSet to specify this. In addition, versions under ios 11 need this to round a view
+     */
```

Comment:
Non sei se é correcto pero algo así?
This function is used version of iOS 11+. In iOS 11 CACornerMask was introduced, with CACornerMask is easier to modify the view layer appeareance